### DISCUSSION TITLE
Fooocus Gallery

### DISCUSSION THREAD

**Discussion by coffeespice** [lllyasviel/Fooocus#1933] [Show and tell]
I made an gallery that allows search prompts in Fooocus Logs  and simplifies dealing with large amount of images and logs.

I hope it is useful for someone else =D

https://github.com/coffeespice/FooocusGallery

![image](https://github.com/lllyasviel/Fooocus/assets/86254525/335ffa44-d2a1-41f7-9aa5-89633f478b0b)

**Comment by 3Diva** [comment]
It looks like a couple of other users have created gallery-type logs for Fooocus images:
https://github.com/lllyasviel/Fooocus/discussions/160#discussioncomment-6772422
https://github.com/lllyasviel/Fooocus/discussions/160#discussioncomment-8126170

I've not tried them, have you? If so, could you tell me how they differ from yours? A log and search for Fooocus images would be neat to have but I don't have the time or space to try the different versions out. No biggy if you haven't tried the others or don't know what the differences in them are, I thought it wouldn't hurt to ask though just in case. 😊

**Reply by coffeespice** [reply]
I didn't knew about other ones.  Gonna check it out. Thanks.

I made this one for my specific need of searching through logs and I`m also analyzing if it is worth integrate it with Instagram.

**Reply by xhoxye** [reply]
It would be nice if it could start with fooocus, or it could merge.

**Comment by parimal718** [comment]
Jst silly question can we see images created by others using fooocus 😬😅 wanna see what people are creating with this ( not getting any information on this topic )

**Reply by coffeespice** [reply]
Civitai has an image tap. That is the only place where I can think of.

**Reply by parimal718** [reply]
what does your gallery does, I mean is it only usable on your private log or we can search other peoples log too

**Reply by coffeespice** [reply]
Only in your private logs. You specify where the logs are, they get indexed and you can search through all your images.

**Reply by coffeespice** [reply]
It is possible to make an public server, so people can upload they logs. But for that work properly the index need improvement.
i'm studding the best way to do it in python (I`m a python developer, btw).

**Comment by PeterJared** [comment]
Anyone tested it? I received a error message when trying to download.

**Reply by coffeespice** [reply]
What error you got?

**Reply by PeterJared** [reply]
![ApplicationFrameHost_QQwBK3jIXK](https://github.com/lllyasviel/Fooocus/assets/49351356/aeb2db33-37ca-4ec4-989e-8a4110593db2)
Default Windows 11 defender
I understand that probably will be  a false error

**Reply by coffeespice** [reply]
Would you please give the name of the file that was blocked? 
My windows defender didn`t alert anything.

**Reply by PeterJared** [reply]
the error appears when trying to download the Zip, then edge cancel the operation
![image](https://github.com/lllyasviel/Fooocus/assets/49351356/300cefbd-ea43-49ae-81f9-940bfc56f157)

**Reply by coffeespice** [reply]
Thanks.

I think the python cached files maybe has something with it.

I removed and ignored them.

**Reply by toutjavascript** [reply]
Seems a good idea. I will try it.
But security is a problem with a python or npm installer

I launched the Fooocus log viewer with only one .html file to download. No danger with corrupted sources.
https://github.com/toutjavascript/Fooocus-Log-Viewer

**Reply by coffeespice** [reply]
I tried this one. It is pretty nice.

In my case search by prompt is a must for me. I do batch generations, so I already have more than 17000 imagens, so things on my side are  a little messy.
